src/sentry/services/hybrid_cloud/organization/service.py

```diff
@@ -274,5 +279,40 @@ def update_option(self, *, organization_id: int, key: str, value: OptionValue) -
     def delete_option(self, *, organization_id: int, key: str) -> None:
         pass
 
+    @regional_rpc_method(resolve=ByOrganizationId())
+    @abstractmethod
+    def send_signal(
+        self, *, organization_id: int, signal: RpcOrganizationSignal, args: Mapping[str, int | None]
+    ) -> None:
+        pass
+
+    def schedule_signal(
+        self,
+        signal: Signal,
+        organization_id: int,
+        args: Mapping[str, int | None],
```

Comment:
Nit: any reason the arg order is different from `send_signal`?

Reply:
Addressed